bi-connectors/PowerBIConnector/OpenSearchProject-ODBC.pq

```diff
@@ -149,7 +149,7 @@ OpenSearchImpl = (Server as text, Port as number, UseSSL as logical) as table =>
             CredentialConnectionString = EncryptedConnectionString
         ],
 
-        FullConnectionString = Diagnostics.LogValue("Connection String", ConnectionString & CredentialConnectionString & EncryptedConnectionString),
+        FullConnectionString = ConnectionString & CredentialConnectionString & EncryptedConnectionString,
```

Comment:
Is there an extra comma or is that correct M? I don't know just asking.

Reply:
With this change have we confirmed that connections still work?

Reply:
There is no extra comma. I have added round brackets around the value to improve readability in https://github.com/Bit-Quill/opensearch-project-sql/pull/27/commits/f27402a988e667b46f62712520cd5337128a3008.
The connection still works with these changes.